test_runner/fixtures/neon_fixtures.py

```diff
@@ -295,10 +295,7 @@ def default_broker(
 ) -> Iterator[NeonBroker]:
     # multiple pytest sessions could get launched in parallel, get them different ports/datadirs
     client_port = port_distributor.get_port()
-    broker_logfile = (
-        get_test_output_dir(request, top_output_dir) / f"storage_broker_{client_port}.log"
-    )
-    broker_logfile.parents[0].mkdir(exist_ok=True, parents=True)
+    broker_logfile = get_test_repo_dir(request, top_output_dir) / "storage_broker.log"
```

Comment:
We can replace the usage of `top_output_dir` session fixture with `test_output_dir` function fixture:

```suggestion
    broker_logfile = test_output_dir / "storage_broker.log"
```

And the logs we don't need `allure_attach_from_dir(Path(broker_logfile))` — logs will be picked up automatically by `test_output_dir` fixture

Reply:
This way, the logs will be outside of the repo dir.
Guess it's fine, will leave it as is for now.

<img width="466" alt="image" src="https://user-images.githubusercontent.com/2690773/208631709-19d8a720-9b4b-4c41-b46b-708bb62c6ea8.png">

Reply:
Feel free to change it to 
```suggestion
    broker_logfile = test_output_dir / "repo" / "storage_broker.log"
```
😄

Reply:
I wanted to avoid this particular hardcode, but if recommended, then sure, will do 😄